Allow multiple files to be passed at once

fantapop opened the pull request:
Previously, only a single path or file name could be passed at once. This was problematic for using crlfmt for tools like lint-staged which expect to be able to pass a list of modified files.  We now support passing any number of files.

rail commented on main.go:
```diff
@@ -80,43 +79,49 @@ func run() error {
-		}
-		if fi.IsDir() {
-			return nil
+	visited := make(map[string]interface{})
```
A nit. `map[string]bool{}` or `map[string]struct{}` would be clearer here.

fantapop replied:
oops yes. I meant to do the struct{} thing.  will fix.

rail reviewed (approved):
LGTM with a nit.
Can I convince you to replace the deprecated `io/ioutil` package with corresponding `os` and `io` functions? :) We require go 1.19, so it's safe to do so.
Something like https://gist.github.com/rail/8ecf4220799cd1740c809079638c1559

fantapop commented:
> LGTM with a nit. Can I convince you to replace the deprecated `io/ioutil` package with corresponding `os` and `io` functions? :) We require go 1.19, so it's safe to do so. Something like https://gist.github.com/rail/8ecf4220799cd1740c809079638c1559

I added an additional commit containing these updates and also removed some unused vars.
